src/arrayMethodShift.js

```diff
@@ -5,7 +5,15 @@
  */
 function applyCustomShift() {
   [].__proto__.shift2 = function() {
-    // write code here
+    this.reverse();
+    let a = this[this.length - 1];
```

Comment:
Please avoid single-letter names

Reply:
Probably this line along with `this.reverse()` and `return a` needs to be moved to `else` block. This way the code readability will increase.

Reply:
also, try to solve it without `this.reverse()`